exercises/practice/pov/tests/pov.rs

```diff
@@ -0,0 +1,421 @@
+/// Reroot a tree so that its root is the specified node.
+mod frompov {
+    use pov::*;
+    use pretty_assertions::assert_eq;
+
+    #[test]
+    fn results_in_the_same_tree_if_the_input_tree_is_a_singleton() {
+        let input = Tree::new("x".to_string());
+        let from = "x".to_string();
+        let result = input.pov_from(&from);
+        let expected = Some(Tree::new("x".to_string()));
+        assert_eq!(
+            result.map(|t| crate::test_util::tree_to_sorted(&t)),
+            expected.map(|t| crate::test_util::tree_to_sorted(&t))
```

Comment:
To make this less verbose, make the `tree_to_sorted` function take an `Option<Tree>` directly. It's only ever used this way. It's not necessarily a beautiful API, but it doesn't matter, because the user doesn't interact with this function. it's more important for the tests themselves to be readable.

Also, import the function at the top of the module:
```rs
mod from_pov {
    use super::test_util::tree_to_sorted;
    // ...
    assert_eq!(tree_to_sorted(result), tree_to_sorted(expected));
```